scripts/leodict.pl

```diff
@@ -4,27 +4,45 @@
 use strict;
 
 use vars qw($VERSION %IRSSI);
-$VERSION = '20040515';
+$VERSION = '20180321';
 %IRSSI = (
-    authors     => 'Stefan \'tommie\' Tomanek',
-    contact     => 'stefan@pico.ruhr.de',
+    authors     => 'Stefan \'tommie\' Tomanek, bw1',
+    contact     => 'bw1@aol.at',
     name        => 'leodict',
     description => 'translates via dict.leo.org',
     license     => 'GPLv2',
     url         => 'http://irssi.org/scripts/',
-    changed     => $VERSION,
     modules     => 'LWP::Simple Data::Dumper',
```

Comment:
Mojo::DOM missing, but if you are going to use Mojo then using the whole mojo suite is just some steps away, use Mojo::UserAgent to fetch the data, say $ua->get('www.perl.org')->result->dom->at('title')->text; result->json gives json output also, not so many dependencies with just cpanm Mojolicious :)
* it's only suggestions so do what you prefer and feel comfortable with :) *